pkg/cloudprovider/vsphere/util.go

```diff
@@ -40,6 +44,10 @@ func GetUUIDFromProviderID(providerID string) string {
 // K8s:    56492e42-22ad-3911-6d72-59cc8f26bc90
 // VMware: 422e4956-ad22-1139-6d72-59cc8f26bc90
 func ConvertK8sUUIDtoNormal(k8sUUID string) string {
+	if len(k8sUUID) < 36 {
```

Comment:
this was probably meant to reference `MinUUIDLen`

Reply:
It was... I even made the const for it... that's what happens when you get sidetracked. 😀